docs/_reporting/hubs/configure-scopes.md

```diff
@@ -32,30 +32,57 @@ FinOps hubs uses Cost Management exports to import cost data for the billing acc
   _Microsoft Cost Management does not support managed exports for Microsoft Customer Agreement billing accounts. Please [configure Cost Management exports manually](#️-configure-exports-manually)._
 </blockquote>
 
-For the most seamless experience, we recommend allowing FinOps hubs to manage exports for you when possible. This is the simplest to setup and requires the least effort to maintain over time. We only recommend configuring Cost Management exports manually if you cannot grant FinOps hubs access to manage exports for you.
+For the most seamless experience, we recommend allowing FinOps hubs to manage exports for you when possible. This option requires the least effort to maintain over time.
+
+<!-- TODO: The  sentence above can be changed when ME can be configured through UI and not command line -->
 
 <br>
 
+
+## 🛠️ Configure exports manually
+
+Manual exports are available if you cannot grant permissions for your scope. This method allows you to create Cost Management exports to accomplish the same goal.
+
+1. [Create a new FOCUS cost export](https://learn.microsoft.com/azure/cost-management-billing/costs/tutorial-export-acm-data?tabs=azure-portal) using the following settings:
+
+   <!-- TODO: Replace the portal link with the docs link when exports v2 docs are available. -->
+
+   - **Type of data** = `Cost and usage details (FOCUS)`<sup>1</sup>
+   - **Dataset version** = `1.0`<sup>2</sup>
+   - **Frequency** = `Daily export of month-to-date costs`<sup>3</sup>
+   - **File partitioning** = On
+   - **Overwrite data** = Off<sup>4</sup>
+   - **Storage account** = (Use subscription/resource deployed with your hub)
+   - **Container** = `msexports`
+   - **Directory** = (Use the resource ID of the scope<sup>5</sup> you're exporting without the first "/")
```

Comment:
Side note: We should revisit this. It's no longer important what the directory path is. In 0.3 and earlier, we used this to identify the scope. In 0.4 we use metadata. Given the new limitations around characters, we might want to simplify the recommendation.

Reply:
https://github.com/microsoft/finops-toolkit/issues/863